packages/kit/src/exports/index.js

```diff
@@ -65,9 +93,11 @@ export function text(body, init) {
 }
 
 /**
- * Generates an `ActionFailure` object.
- * @param {number} status
- * @param {Record<string, any> | undefined} [data]
+ * Create an `ActionFailure` object.
+ * @template {Record<string, any> | undefined} T
```

Comment:
This defaults the `T` to `undefined` if not set which makes it equivalent to the `d.ts` definition. Maybe that fixes the bug you asked about three lines below.
```suggestion
 * @template {Record<string, any> | undefined} [T=undefined]
```

Reply:
Mhm doesn't seem to have changed the test failure outcome, but the change is still correct.